mackup/applications/git.cfg

```diff
@@ -3,3 +3,7 @@ name = Git
 
 [configuration_files]
 .gitconfig
+
+[xdg_configuration_files]
+git/config
+git/credentials
```

Comment:
Fair enough, although they aren't keys...

The entirety of mine is:
```
[credential]
	helper = osxkeychain

[user]
	name = William Myers
        email=
```

It is usually used so people can share .gitconfig files and keep their emails etc. separate...

But I completely understand if you just want to do:

```
[xdg_configuration_files]
git/ignore
```